shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -670,12 +672,12 @@ - (void)updateSemanticsCustomActions:(const FlutterSemanticsCustomAction*)action
     // Attaches the accessibility root to the flutter view.
     auto root = _bridge->GetFlutterPlatformNodeDelegateFromID(0).lock();
     if (root) {
-      if ([self.viewController.view.accessibilityChildren count] == 0) {
+      if ([self.viewController.flutterView.accessibilityChildren count] == 0) {
         NSAccessibilityElement* native_root = root->GetNativeViewAccessible();
-        self.viewController.view.accessibilityChildren = @[ native_root ];
+        self.viewController.flutterView.accessibilityChildren = @[ native_root ];
       }
     } else {
-      self.viewController.view.accessibilityChildren = nil;
+      self.viewController.flutterView.accessibilityChildren = nil;
```

Comment:
Ditto here - we should avoid loading the view if it isn't already loaded. 

That said, I'm not clear on what should happen if it's not loaded and you want to set its children - perhaps you want some kind of assertion there, but it should probably be covered whenever the view does load so that we set the children then rather than forcing it to load to set them here.